### PR TITLE
[69] [Chore] Add UI test for Home Screen

### LINKED ISSUE #69: [Chore] Add UI test for Home Screen
## Why

- Home Screen needs UI tests
 
## Acceptance Criteria

- Add test functions for Home screen
- Tests passed

### PR DESCRIPTION
**Pull request by chiiida**
https://github.com/llleyelll/ic-surveys-ios/issues/69

## What happened 👀

- Add UI test for Home Screen
 
## Proof Of Work 📹

Screen Shot 2021-07-14 at 11 16 48 AM

### CODE REVIEW

File: SurveyApp/Modules/Home/Cell/SurveyCollectionCell.swift [hunk 1]
```diff
@@ -78,4 +79,8 @@ final class SurveyCollectionCell: UICollectionViewCell {
         descriptionLabel.font = UIFont.regular(ofSize: .body)
         descriptionLabel.numberOfLines = 2
     }
+    
+    private func setIdentifier() {
```
Comment (markgravity) [inline SurveyApp/Modules/Home/Cell/SurveyCollectionCell.swift]:
```suggestion
    private func setIdentifiers() {
```

**Review by markgravity — APPROVED** [review]
LGTM 🤖

File: SurveyApp/Modules/Splash/SplashInteractor.swift [hunk 2]
```diff
@@ -32,6 +34,7 @@ final class SplashInteractor {
 extension SplashInteractor: SplashInteractorInput {
     
     var isLoggedIn: Bool {
-        userSessionProvider?.isLoggedIn ?? false
+        guard ProcessInfo.processInfo.environment[TestConstants.userLoggedInKey] == nil else { return true }
```
Comment (blyscuit) [inline SurveyApp/Modules/Splash/SplashInteractor.swift]: let's make a Helper class for this like
```
enum UITestConfiguration {
static var isLoggedIn { 
 return ProcessInfo.processInfo.environment[TestConstants.userLoggedInKey] != nil 
 }
}

**Review by blyscuit — APPROVED** [review]
Looks good, one comment.

File: SurveyApp/Modules/Splash/SplashInteractor.swift (hunk 2 above)
Comment (chiiida) [reply SurveyApp/Modules/Splash/SplashInteractor.swift]: I updated the code. Thank you 🙏

**Pull request merged**